examples/heterogeneous-node-classification-with-hashgnn.ipynb

```diff
@@ -282,7 +282,7 @@
    "source": [
     "# Add a HashGNN node property step to the pipeline\n",
     "_ = pipe.addNodeProperty(\n",
-    "    \"beta.hashgnn\",\n",
+    "    \"hashgnn\",\n",
```

Comment:
This notebook has a text at the top which says GDS must be `1.6`. With this change it must be `2.5`.

Reply:
https://github.com/neo-technology/graph-analytics/blob/293a1ba53b88aa9e138f085ac32fbcd574668f7e/release/archive/2.5.0.md#L207